src/map/battle.cpp

```diff
@@ -7193,8 +7192,9 @@ static void battle_calc_attack_plant(struct Damage* wd, struct block_list *src,s
 		return;
 	}
 
-	// Triple Attack and Finger Offensive have a special property, they do not split damage on plant mode
+	// Skills have a special property, they do not split damage on plant mode
 	// In pre-renewal, it requires the monster to have exactly 100 def
+	std::shared_ptr<s_skill_db> skill = skill_db.find(skill_id);
 	if (skill != nullptr && skill->inf2[INF2_MULTIHITONPLANTS] && wd->div_ < 0
```

Comment:
```suggestion
	if (std::shared_ptr<s_skill_db> skill = skill_db.find(skill_id); skill != nullptr && skill->inf2[INF2_MULTIHITONPLANTS] && wd->div_ < 0
```
This makes the variable only available inside the if().